tools/packaging/kata-deploy/local-build/cache/Dockerfile

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2023 Intel Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+
+FROM busybox:musl
```

Comment:
why the musl built busybox? Asking to understand if consumers can face any kind of issue due glibc vs musl